.travis.yml

```diff
@@ -1,27 +1,32 @@
 language: php
 
-notifications:
-  email:
-    on_success: never
-    on_failure: change
-
 branches:
   only:
     - master
 
 php:
-  - 5.3
-  - 5.6
+  - '5.5'
+  - '5.6'
+  - '7.0'
 
 env:
   - WP_VERSION=latest WP_MULTISITE=0
+  - WP_VERSION=latest WP_MULTISITE=1
+  - WP_VERSION=4.6 WP_MULTISITE=0
 
 matrix:
-  include:
-    - php: 5.3
-      env: WP_VERSION=latest WP_MULTISITE=1
+  exclude:
+    - php: '7.0'
+      env: WP_VERSION=4.2 WP_MULTISITE=0
 
 before_script:
+  - nvm install 4.2
   - bash bin/install-wp-tests.sh wordpress_test root '' localhost $WP_VERSION
+  - composer install
+  - npm install -g gulp
+  - npm install
 
-script: phpunit
+script:
+  - gulp lint
+  - phpunit -c codecoverage.xml.dist
+  - phpunit -c multisite.xml.dist
```

Comment:
we could perhaps only run the codecoverage file to avoid running the tests twice. Travis handles multisite runs using the WP_MULTISITE env variable.